Project Template/README.md

```diff
@@ -0,0 +1,49 @@
+# VMware Project Template in Visual Studio for .NET Automation SDK
+## Table of Contents
+- [Abstract](#abstract)
+- [Supported vCenter Releases](#supported-vcenter-releases)
+- [Table of Contents](#table-of-contents)
+- [Quick Start Guide](#quick-start-guide)
+  - [Setting up build environment](#setting-up-build-environment)
+  - [Setting up a vSphere Test Environment](#setting-up-a-vsphere-test-environment)
+  - [Building the Samples](#building-the-samples)
+
+## Abstract
+This document describes how to import a VMWare Automation Project Template to Visual Studio 
+which has been developed to work with .NET Framework 4.5.
+With this template you will be able to have a basic structure to add your custom code and interact with the vAPI.
+
+## Supported vCenter Releases:
+vCenter 6.0 and 6.5 and vCenter 6.6.1 for VMware Cloud 1.1.
+
+Please refer to the notes in each sample for detailed compatibility information.
+
+## Quick Start Guide
+This document will walk you through getting up and running with the import a VMware Project Template in Visual Studio for .NET Automation SDK.
+
+### Setting up build environment
+To import the VMware Project Template, install Microsoft Visual Studio 2015 or higher. 
+
+* Copy the zip file (VMWare-SDK-Automation-Sample-Template.zip) into the the folder C:\Users\<username>\Documents\Visual Studio 2015\Templates\ProjectTemplates
+
+### Setting up a Project Template Test Environment
```

Comment:
Why do you need vSphere test environment for the template ? This is not required. Please keep the instructions minimal. We already have detailed instructions in the main README.